routes/web.php

```diff
@@ -36,6 +36,9 @@
         Route::get('/events',[AdminController::class,'showEvents'])->name('showEvents');
         Route::get('/events/create',[AdminController::class,'createEvent'])->name('createEvent');
         Route::post('/events/store',[AdminController::class,'storeEvent'])->name('storeEvent');
+        Route::get('/events/edit/{event}',[AdminController::class,'editEvent'])->name('editEvent');
+        Route::patch('/events/update/{id}',[AdminController::class,'updateEvent'])->name('updateEvent');
```

Comment:
there's no need to add the word `update` in the route url since we have indicated the `patch` method already. please refer to the documentation `https://laravel.com/docs/10.x/controllers#creatable-singleton-resources`

Reply:
So should I change from "/event/update/{id}" to "/events/{id}"?

Reply:
@JuriMino yes but I suggest to use {event} instead of {id}
If you can create an `events` route group it would be better also